### PR TITLE
[T4A7][T17-B1]Chia Zhe Min

### PR DESCRIPTION
**Pull request by zhems**

### CODE REVIEW

**Comment by zhems** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Name.java [hunk 1]
```diff
@@ -60,5 +60,19 @@ public boolean equals(Object other) {
     public int hashCode() {
         return fullName.hashCode();
     }
+    
+    /**
+     * Returns true of the other name is very similar to this name.
+     * Two names are considered similar if they have the exact same characters in the same order, regardless of case
```
Comment (JoelT-92) [inline src/seedu/addressbook/data/person/Name.java:66]: Well done with the header comment!

File: test/java/seedu/addressbook/person/NameTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class NameTest {
+
+    @Test
+    public void sameNameButDifferentCase_returnTrue() throws IllegalValueException {
+        Name james = new Name("james");
```
Comment (JoelT-92) [inline test/java/seedu/addressbook/person/NameTest.java:18]: Similar code below, so you can refactor this into another method.

**Comment by JoelT-92** [comment]
@zhems Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Name.java [hunk 3]
```diff
@@ -60,5 +60,13 @@ public boolean equals(Object other) {
     public int hashCode() {
         return fullName.hashCode();
     }
+    
+    /**
+     * Returns true of the other name is very similar to this name.
+     * Two names are considered similar if ...
+     */
+     public boolean isSimilar(Name other) {
```
Comment (JoelT-92) [inline src/seedu/addressbook/data/person/Name.java:68]: First commit should be a commit showing your first test case : Function not yet implemented.

**Pull request closed without merging**